hazelcast/src/test/java/com/hazelcast/client/cache/instance/ClientCacheThroughHazelcastInstanceTest.java

```diff
@@ -64,7 +64,8 @@ protected HazelcastInstance createInstance(Config config) {
         ownerInstance = instanceFactory.newHazelcastInstance(config);
 
         ClientConfig clientConfig = new ClientConfig();
-        clientConfig.getConnectionStrategyConfig().getConnectionRetryConfig().setClusterConnectTimeoutMillis(1000);
+        clientConfig.getConnectionStrategyConfig().getConnectionRetryConfig().setClusterConnectTimeoutMillis(1_000);
```

Comment:
Just curious: what was the motivation for this change? Can't this lead to more frequent test failures in case of hiccups?

Reply:
This is what helped to find the race in the client shutdown. Also, it decreased test speed significantly. But I agree that it may lead to flaky failures in case of hiccups. So I increased the value to 10s. This is smaller than before (30s), but should be enough to avoid flakiness.